docs/IDL_other.adoc

```diff
@@ -0,0 +1,186 @@
+= PIG — Interface Definition Language
+:toc:
+:toc-title: Page Contents
+:sectlinks:
+
+IMPORTANT: This specifically does not use an Ion DSL.
```

Comment:
I only include this to spark discussion on possible benefits of of not using an Ion DSL.